lib/domain/entities/hardware_platform.dart

```diff
@@ -1,143 +1,143 @@
 import 'package:cobble/ui/common/icons/watch_icon.dart';
 
 enum PebbleHardwarePlatform {
-  UNKNOWN,
-  PEBBLE_ONE_EV1,
-  PEBBLE_ONE_EV2,
-  PEBBLE_ONE_EV2_3,
-  PEBBLE_ONE_EV2_4,
-  PEBBLE_ONE_POINT_FIVE,
-  PEBBLE_TWO_POINT_ZERO,
-  PEBBLE_SNOWY_EVT2,
-  PEBBLE_SNOWY_DVT,
-  PEBBLE_BOBBY_SMILES,
-  PEBBLE_ONE_BIGBOARD_2,
-  PEBBLE_ONE_BIGBOARD,
-  PEBBLE_SNOWY_BIGBOARD,
-  PEBBLE_SNOWY_BIGBOARD_2,
-  PEBBLE_SPALDING_EVT,
-  PEBBLE_SPALDING_PVT,
-  PEBBLE_SPALDING_BIGBOARD,
-  PEBBLE_SILK_EVT,
-  PEBBLE_SILK,
-  PEBBLE_SILK_BIGBOARD,
-  PEBBLE_SILK_BIGBOARD_2_PLUS,
-  PEBBLE_ROBERT_EVT,
-  PEBBLE_ROBERT_BIGBOARD,
-  PEBBLE_ROBERT_BIGBOARD_2
+  unknown,
+  pebbleOneEv1,
+  pebbleOneEv2,
+  pebbleOneEv2_3,
+  pebbleOneEv2_4,
+  pebbleOnePointFive,
+  pebbleOnePointZero,
+  pebbleSnowyEvt2,
+  pebbleSnowyDvt,
+  pebbleBobbySmiles,
+  pebbleOneBigboard2,
+  pebbleOneBigboard,
+  pebbleSnowyBigboard,
+  pebbleSnowyBigboard2,
+  pebbleSpaldingEvt,
+  pebbleSpaldingPvt,
+  pebbleSpaldingBigboard,
+  pebbleSilkEvt,
+  pebbleSilk,
+  pebbleSilkBigboard,
+  pebbleSilkBigboard2Plus,
+  pebbleRobertEvt,
+  pebbleRobertBigboard,
+  pebbleRobertBigboard2
 }
 
 PebbleHardwarePlatform pebbleHardwarePlatformFromNumber(int number) {
   switch (number) {
     case 1:
-      return PebbleHardwarePlatform.PEBBLE_ONE_EV1;
+      return PebbleHardwarePlatform.pebbleOneEv1;
```

Comment:
ditto TimelinePinLayout

Reply:
These numbers must be constant since they correspond to integers that get sent to the watch. I don't think generating them is a good idea since generator can change values.

Reply:
What about this:
```
class PebbleHardwarePlatform {
  final int id;

  const PebbleHardwarePlatform._(this.id);

  static const pebbleOneEv1 = PebbleHardwarePlatform._(1);
  static const pebbleOneEv2 = PebbleHardwarePlatform._(2);

  static const values = [pebbleOneEv1, pebbleOneEv2];

  static PebbleHardwarePlatform fromNumber(int number) {
    final platform = values.firstWhere(
      (element) => element.id == number,
      orElse: () => null,
    );
    assert(
      platform != null,
      'Unknown hardware platform $number',
    );
    return platform;
  }

  int toNumber() => this.id;
}
```
you would use it similarly:
```
return PebbleFirmware(
  [...]
  PebbleHardwarePlatform.fromNumber(pigeon.hardwarePlatform),
  [...]
);
```